components/event-sources/reconciler/httpsource/reconcile.go

```diff
@@ -104,7 +113,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
 		return err
 	}
 
-	return r.syncStatus(src, ch, ksvc)
+	policy, err := r.reconcilePolicy(src, ksvc)
+	if err != nil {
+		return err
+	}
+	return r.syncStatus(src, ch, ksvc, policy)
```

Comment:
(nit) A new line would be appropriate here, because `syncStatus` is not directly related to the `reconcilePolicy` logic.

Reply:
Added a new line